packages/access-client/src/utils/buffers.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert a Uint8Array to an ArrayBuffer, taking into account
+ * that we may be looking at a "data view".
+ * thanks, https://stackoverflow.com/a/54646864
+ * 
+ * If we aren't looking at a data view, simply returns the underlying ArrayBuffer
+ * directly.
+ *
+ * @param {Uint8Array} array
+ * @returns ArrayBuffer
+ */
+export function uint8ArrayToArrayBuffer(array) {
```

Comment:
btw I don't understand why this can't just return `array.buffer`

If the point is explicitly to copy to a new array, then perhaps the jsdoc should say that (and maybe the function name should have 'copy' in it?)

Reply:
like maybe this could be called `copyTypedArray`

Also if the point is to accomodate a `DataView` maybe the type of the `array` param should be `DataView`?

Reply:
travis corrected me that `Uint8Array#buffer` may actually refer to a much wider buffer